server/utils/middleware.js

```diff
@@ -0,0 +1,60 @@
+const logger = require('./logger')
+
+const requestLogger = (request, _, next) => {
```

Comment:
You probably may know this, but there is an npm package called `logger` that does this exact thing you're doing here. You can find it here https://www.npmjs.com/package/logger